src/internal/context.go

```diff
@@ -17,9 +17,11 @@ type Context struct {
 	Image      string
 	Build      DockerBuild
 	Env        map[string]string
+	HostEnv    []string `yaml:"hostenv,omitempty"`
```

Comment:
comment meant for line 15: shouldn't it be `yaml:"-"` since the context name is not in the yaml file but built by us? Doesn't matter though but odd.